lab-doug/app/component/create-list-form/index.js

```diff
@@ -0,0 +1,32 @@
+'use strict';
+require('./create-list-form.scss');
+const angular = require('angular');
+
+angular.module('widgetApp').directive('createListForm', function(){
+  return {
+    restrict: 'E',
+    replace: true,
+    template: require('./create-list-form.html'),
+    controller: ['$log', 'listService', CreateListFormController],
+    controllerAs: 'createListFormCtrl',
+    bindToController: true,
+    scope: {}
+  };
+});
+
+function CreateListFormController($log, listService){
+
+  $log.debug('entered CreateListFormController');
+  this.list = {};
+  this.createList = function(){
+    $log.debug('entered createListFormCtrl.createList');
+    listService.createList(this.list)
+    .then(() => {
+      this.list = {};
+    })
+    .catch(() => {
```

Comment:
This doesn't seem great for debugging -- how do you know whether you've had an error or not?